src/form/bf_mx_lin.jl

```diff
@@ -92,36 +126,45 @@ function constraint_mc_load_power_balance(pm::LPUBFDiagModel, nw::Int, i, bus_ar
     ps   = get(var(pm, nw),   :ps, Dict()); _PM._check_var_keys(ps, bus_storage, "active power", "storage")
     qs   = get(var(pm, nw),   :qs, Dict()); _PM._check_var_keys(qs, bus_storage, "reactive power", "storage")
 
+
     cstr_p = []
     cstr_q = []
 
-    cstr_p = JuMP.@constraint(pm.model,
-        sum(p[a] for a in bus_arcs)
-        + sum(psw[a_sw] for a_sw in bus_arcs_sw)
-        + sum(pt[a_trans] for a_trans in bus_arcs_trans)
-        .==
-        sum(pg[g] for g in bus_gens)
-        - sum(ps[s] for s in bus_storage)
-        - sum(pd[d] for d in bus_loads)
-        - sum(gs.*w for gs in values(bus_gs))
-    )
-
-    cstr_q = JuMP.@constraint(pm.model,
-        sum(q[a] for a in bus_arcs)
-        + sum(qsw[a_sw] for a_sw in bus_arcs_sw)
-        + sum(qt[a_trans] for a_trans in bus_arcs_trans)
-        .==
-        sum(qg[g] for g in bus_gens)
-        - sum(qs[s] for s in bus_storage)
-        - sum(qd[d] for d in bus_loads)
-        + sum(bs.*w for bs in values(bus_bs))
-    )
-
-    con(pm, nw, :lam_kcl_r)[i] = cstr_p
-    con(pm, nw, :lam_kcl_i)[i] = cstr_q
+   cnds = conductor_ids(pm; nw=nw)
+   ncnds = length(cnds)
+
+   for c in 1:ncnds
+     if(vmax[c]!=0)
+       cstr_p = JuMP.@constraint(pm.model,
+        sum((p[a])[c]*1000.0 for a in bus_arcs)
+        + sum((psw[a_sw])[c]*1000.0 for a_sw in bus_arcs_sw)
+        + sum((pt[a_trans])[c]*1000.0 for a_trans in bus_arcs_trans)
+        == 
+        sum((pg[g])[c]*1000.0 for g in bus_gens)
+        - sum((ps[s])[c]*1000.0 for s in bus_storage)
+        - sum((pd[d])[c]*1000.0 for d in bus_loads) 
+	- sum(gs[c]*w[c]*1000.0 for gs in values(bus_gs)))
+    end
+    end
+
+
+    for c in 1:ncnds
+      if(vmax[c]!=0)
+       cstr_q = JuMP.@constraint(pm.model,
+        sum((q[a])[c]*1000.0 for a in bus_arcs)
+        + sum((qsw[a_sw])[c]*1000.0 for a_sw in bus_arcs_sw)
+        + sum((qt[a_trans])[c]*1000.0 for a_trans in bus_arcs_trans)
+        ==
+        sum((qg[g])[c]*1000.0 for g in bus_gens)
+        - sum((qs[s])[c]*1000.0 for s in bus_storage)
+        - sum((qd[d])[c]*1000.0 for d in bus_loads)
+        + sum(bs[c]*w[c]*1000.0 for bs in values(bus_bs)))
+   end
+   end
```

Comment:
I'm not sure of the purpose behind this change, previously it was automatically cleanly over all phases, but now there is a loop over all phases? If the purpose is to eliminate phases which were padded/projected, and not in the original data, I think this isn't really the best way to accomplish that